src/components/Breadcrumb/Breadcrumb.test.jsx

```diff
@@ -39,8 +36,8 @@ describe('Breadcrumb with overflow', () => {
   });
 
   afterAll(() => {
-    Object.defineProperty(HTMLElement.prototype, 'clientWidth', originalOffsetHeight);
-    Object.defineProperty(HTMLElement.prototype, 'scrollWidth', originalOffsetWidth);
+    delete HTMLElement.prototype.clientWidth;
```

Comment:
doesn't this potentially break downstream tests?

Reply:
Not anymore. In JSDOM v16 they [no longer share prototypes](https://github.com/jsdom/jsdom/blob/master/Changelog.md#1600)

> Each jsdom Window now creates new instances of all the web platform globals. That is, our old [shared constructor and prototypes caveat](https://github.com/jsdom/jsdom/blob/35894a6703ed1f4de98942780bd99244ac27f600/README.md#shared-constructors-and-prototypes) is no longer in play.